R/bundle.R

```diff
@@ -263,7 +288,9 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters, a
   metadata <- list(appmode = appMode)
 
   # emit appropriate primary document information
-  primaryDoc <- ifelse(is.null(appPrimaryDoc), NA, appPrimaryDoc)
+  primaryDoc <- ifelse(is.null(appPrimaryDoc) ||
```

Comment:
No, this code is temporary, and exists to maintain the facade for Connect that this is just a run-of-the mill Shiny app. In the future we'll always write the `primaryDoc` to the manifest, and Connect will use it to form the argument to `runApp`.